sdk/src/main/kotlin/io/rover/sdk/ui/blocks/poll/image/ImagePollOptionView.kt

```diff
@@ -231,12 +238,22 @@ internal class ImagePollOptionView(context: Context?) : RelativeLayout(context)
             )
 
             if (isSelectedOption) {
+
                 voteIndicatorView.measure(0, 0)
                 maxWidth =
                     viewWidth - voteIndicatorView.measuredWidth - (bottomSectionHorizontalMargin * 2) - (voteIndicatorView.layoutParams as MarginLayoutParams).marginStart
             }
         }
 
+        isClickable = false
+
+        contentDescription = if (isSelectedOption) {
+            "Your vote, ${option.text.rawValue}, has $votingShare percent of the votes"
+        } else {
+            "${option.text.rawValue}, has $votingShare percent of the votes"
+        }
```

Comment:
I think we can make the text a little more concise here (a11y users seem to be very sensitive to time and thus verbosity in my experience just because of the amount of crap they have to sift through serially; for example they often have their TTS engine turned up to LUDICROUS SPEED).

So perhaps instead of "[Your vote] $optionText has $votingShare percent of the votes". just do:

    "[Your vote] $optionText, $votingShare percent".